### ISSUE TITLE
Course 2: Hyperparameter tuning, Batch Normalization and Programming Frameworks: Week 2

### ISSUE THREAD

**Issue by mbhushan** [mbhushan/ml#1]
<img width="456" alt="screen shot 2018-01-07 at 7 44 20 am" src="https://user-images.githubusercontent.com/898047/34650987-bd865aa6-f37e-11e7-94c5-a4a1eab488d8.png">
